realsense2_camera/src/realsense_node.cpp

```diff
@@ -191,6 +192,11 @@ void RealSenseNode::getDevice() {
             return;
         }
 
+        // make sure the hardware is initialized correctly by
```

Comment:
Not sure if this comment should have more words in it, to me it sounds like it stops mid-sentance.
Also whitespace below.